src/Microsoft.AspNetCore.Server.Kestrel/Http/Frame.cs

```diff
@@ -1097,13 +1079,6 @@ protected void ReportApplicationError(Exception ex)
             Log.ApplicationError(ConnectionId, ex);
         }
 
-        private enum HttpVersionType
```

Comment:
I prefer the more "normalized" version of having the enum instead of a string and a bool.

This makes it clear that there are only 2 or 3 valid states.